README.md

```diff
@@ -1,10 +1,8 @@
 # Stashbot
 
-Bitbucket Server 4.X Build Status: [![Build Status](https://travis-ci.org/terabyte/stashbot.svg?branch=master)](https://travis-ci.org/terabyte/stashbot)
+Dwolla has forked this plugin to upgrade it to Bitbucket 5.  Future upgrades are unlikely.  Responses to pull requests and issues may be slow.  
```

Comment:
For PR purposes I would split this into a separate Dwolla-customizations commit

Reply:
Done.  Thanks for this feedback.